.github/workflows/release.yml

```diff
@@ -25,6 +24,10 @@ jobs:
           go-version: '~1.22'
       - name: Clean dist directory
         run: rm -rf dist || true
+      - name: Install Syft to generate SBOMs
+        run: |
+          curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b $HOME/bin
```

Comment:
is it worth to consider caching the binary?

Reply:
I do not think so.
We just run this action when we do the release (not so many times per year)